cloudhub-2/modules/ROOT/pages/ch2-comparison.adoc

```diff
@@ -57,6 +57,7 @@ CloudHub 2.0 application behavior deviates from CloudHub 1.0 in the following wa
 * HTTP, HTTPS, and TCP inbound protocols are supported. Inbound protocols that are not TCP or HTTP-based are not supported.
 * In xref:monitoring::alerts.adoc[Anypoint Monitoring], you must set alerts for apps individually. Setting alerts for all apps simultaneously is not supported.
 * CloudHub application workers and CloudHub 2.0 application replicas of the same vCore sizes may differ in performance metrics and cannot be compared.
+* The grace period for CloudHub applications is two minutes, whereas, the grace period for CloudHub 2.0 applications is five minutes. For more information, see: xref:ch2-deploy.adoc#grace-period[Grace Period].
```

Comment:
I suggest avoiding whereas, but if you use it, you should remove the second comma. Also, the colon between see and the Grace Period link should be removed. 

writer.ai suggests the following (among other variations): 
CloudHub applications have a two-minute grace period. CloudHub 2.0 applications have a five-minute grace period. For more information, see xref:ch2-deploy.adoc#grace-period[Grace Period].